common/src/main/java/com/microsoft/identity/common/internal/cache/CacheKeyValueDelegate.java

```diff
@@ -163,6 +164,8 @@ public String generateCacheKey(Credential credential) {
             }
 
             cacheKey = cacheKey.replace(CLIENT_ID, familyIdForCacheKey);
+        } else if (credential instanceof PrimaryRefreshTokenRecord) {
+            cacheKey = cacheKey.replace(CLIENT_ID, "");
```

Comment:
Hi @oldalton, this is cache key for PRT.
Now it only constructed from HOME_ACCOUNT_ID, ENVIRONMENT and CREDENTIAL_TYPE.
Tests add `primaryRefreshTokenCacheKeyComplete`

Reply:
Hi, please follow the schema as described here: https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FSSO%2FSchema.md (Primary refresh token schema section)

Key should contain home account id, environment, credential type, and clientId in a specific order.

Reply:
Thanks, please help review again..